src/hw/hw_mngr.cpp

```diff
@@ -13,10 +13,136 @@
 
 #include <charconv>
 #include <filesystem>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+#include <set>
+#include <vector>
 
 namespace fs = std::filesystem;
 using namespace phosphor::logging;
 
+namespace fans
```

Comment:
Is it possible to move the logic of working with fans into a separate class and file?

Reply:
I am not sure that is needed now.